setup.py

```diff
@@ -103,10 +129,53 @@ def get_version():
             'Programming Language :: Python :: 3.6',
         ],
         license='Apache License 2.0',
-        setup_requires=['pytest-runner'],
+        setup_requires=['pytest-runner', 'cython', 'torch>=1.1', 'numpy'],
```

Comment:
We may assume that users have already installed PyTorch before they install mmdetection. Sometimes the installed pytorch cannot be identified correctly during the installation of mmdetection, so that adding torch in `setup_requires` may lead to unexpected behavior.

Reply:
OK, I wanted to add the version requirement as having 1.0 installed leads to linker errors that are not easy to debug